packages/openhim-core-api/src/jembi-openhim-core-api.ts

```diff
@@ -114,17 +114,91 @@ export async function getImportMap(): Promise<any> {
   return response.data
 }
 
+export async function fetchRoles(): Promise<any> {
+  await ensureApiClientInitialized()
+  const response = await apiClient.get('/roles')
+  return response.data
+}
```

Comment:
Based on the discussion we had for client role. We should probably be fetching this from clients and not the roles endpoint.